src/test/exception_translator_test.cpp

```diff
@@ -114,42 +114,50 @@ BOOST_AUTO_TEST_CASE( empty_translator_translates_standard_exception )
 {
     bpe::exception_translator translator;
 
-	try {
-		translate_index_error(translator);
-		BOOST_FAIL("No exception thrown");
-	} catch (bpe::exception const & error) {
-		std::string const expected_message("exceptions.IndexError: list index out of range");
-		BOOST_CHECK_EQUAL(expected_message, error.what());
-	}
+    try {
+        translate_index_error(translator);
+        BOOST_FAIL("No exception thrown");
+    } catch (bpe::exception const & error) {
+        std::ostringstream expected_message;
+#if PY_MAJOR_VERSION == 2
```

Comment:
This could be replaced by a function which generates the qualified exception name given an unqualified one. This would reduce the number of macros in actual test code, plus it would be less confusing.